server/util/twilio.js

```diff
@@ -5,6 +5,10 @@ import Twilio from 'twilio';
 const twilioClient = new Twilio();
 const CALLER_ID = process.env.TWILIO_CALLER_ID;
 
-// Added the below console.logs to pass linting - please delete!
-console.log(twilioClient);
-console.log(CALLER_ID);
+export function sayHelloUser(userFirstName, campaignName) {
```

Comment:
We need to test this method and make sure it returns the expected twiml when passed a user name and campaign name.